api/api/models/base_user_owned_model.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from .. import settings
+from .base_model import BaseModel
+
+
+class BaseUserOwnedModel(BaseModel):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, to_field="id")
+
+    class Meta:
+        abstract = True
```

Comment:
We can inherit from this class when we want to create a new model in the database that is owned by the user